src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -3719,6 +3735,54 @@ def webapp_up(cmd, name, resource_group_name=None, plan=None, location=None, sku
     return create_json
 
 
+def _update_app_settings_for_windows_if_needed(cmd, rg_name, name, match, site_config, runtime_version):
```

Comment:
Fixes bug where running `az webapp up` using an existing app, but having a different runtime doesn't actually change the runtime